build.py

```diff
@@ -9,13 +9,16 @@
 
     os.environ["CONAN_BUILD_POLICY"] = "missing"
     
-    builder = build_template_default.get_builder()
-    
+    builder = build_template_default.get_builder(pure_c=False)
+
+    # Revert this change once boost is available for other than libstdc++11.    
     modified_builds = []
     for settings, options, env_vars, build_requires, reference in builder.items:
-        options["gflags:nothreads"] = False
+        if settings.get("compiler.libcxx") == "libstdc++":
```

Comment:
if it doestn't support some profile, we need to raise an exception on `configure`